### PR TITLE
Use dotfile format for metadata

### PR DESCRIPTION
**Pull request by SamCarlberg**
See https://github.com/wpilibsuite/allwpilib/pull/600

### CODE REVIEW

File: src/main/java/edu/wpi/first/smartdashboard/livewindow/elements/LWSubsystem.java [hunk 1]
```diff
@@ -86,7 +86,7 @@ private void addSubsystemElement(String key, ITable value) {
       System.out.println(
           "\nSubsystem \"" + getFieldName() + "\" does not contain widget \"" + key + "\"");
       System.out.println("Table: " + value);
-      System.out.println("Type: " + value.getString("~TYPE~", ""));
+      System.out.println("Type: " + value.getString(".type", null));
```
Comment (AustinShalit) [inline src/main/java/edu/wpi/first/smartdashboard/livewindow/elements/LWSubsystem.java]: Why the change from `""` to `null`?

Comment (SamCarlberg) [inline src/main/java/edu/wpi/first/smartdashboard/livewindow/elements/LWSubsystem.java]: More descriptive than just an empty string

**Comment by AustinShalit** [comment]
Waiting on wpilibsuite/allwpilib#600
